sale_commission/models/sale_commission_mixin.py

```diff
@@ -66,11 +66,12 @@ def _compute_commission_status(self):
                     "%s commission agents"
                 ) % len(line.agents)
 
-    def _prepare_agents_vals(self):
+    def _prepare_agents_vals(self, vals=None):
         """Hook method for preparing the values of agents.
 
         :param: self: Record of the object that is being handled.
         """
+        assert self or vals
```

Comment:
Use orm exception instead

Reply:
This is a developer assert, not a user one. That's why I put it that way.